.github/workflows/mobilecoin-dispatch-dev-deploy.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+#
+# MobileCoin Core projects - Dispatch (manual) Job - Deploy core apps to the development namespace.
+
+name: mobilecoin-dispatch-dev-deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      namespace:
+        description: "Target Namespace"
+        type: string
+        required: true
+      version:
+        description: "Chart Version"
+        type: string
+        required: true
+      docker_image_org:
+        description: "Docker Image Org"
+        type: string
+        required: true
```

Comment:
I was going to ask why have `required` and `default`, but it looks like `required` is just a _communication_ mechanism between authors and users,  https://github.com/actions/runner/issues/1070

Reply:
Well I thought that "required" did something, but maybe not :shrug:   

Although we set a default, we still want to make sure a user puts _something_ in this value. Having an empty docker image org value would break the workflow.